<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Updater</class>
 <widget class="QMainWindow" name="Updater">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Updater</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-7</y>
      <width>401</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(100, 100, 100);</string>
       </property>
       <property name="text">
        <string>Check for updates..</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="downloadProgressBar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>4</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>4</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
	border: 1px solid #000000;
	background-color: #222222;
}

QProgressBar::chunk {
	background-color: rgb(224, 27, 36);
	width: 1px;
}</string>
       </property>
       <property name="value">
        <number>13</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="overallProgressBar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>4</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>4</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
	border: 1px solid #000000;
	background-color: #222222;
}

QProgressBar::chunk {
	background-color: rgb(224, 27, 36);
	width: 1px;
}</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="checkButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: #C82828;
	border-radius: 0;
	border: 1px solid #000;
	color: rgb(0, 0, 0);
}

QPushButton:hover {
	background-color: #D82828;
}</string>
         </property>
         <property name="text">
          <string>Check files</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="optionButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: #2d2d2d;
	border-radius: 0;
	border: 1px solid #000;
	color: #C82828;
}

QPushButton:hover {
	background-color: #3d3d3d;
}</string>
         </property>
         <property name="text">
          <string>Options</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="launchButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-radius: 0;
	background-color: #000000;
	color: #C82828;
	border: 1px solid #242424;
}

QPushButton:hover {
	background-color: #1f1f1f;
}
</string>
         </property>
         <property name="text">
          <string>Launch game</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
